src/component/tag/tag.scss

```diff
@@ -1,26 +1,17 @@
 @import '@/style/fonts.scss';
 @import '@/style/theme.scss';
 
-$colors: (
-  'primary-color': 'background-primary-variant',
-  'primary-color-variant-3': 'background-primary-variant-3',
-  'primary-color-variant-4': 'background-primary-variant-4'
-);
-
 .okp4-dataverse-portal-tag-main {
-  @include with-theme {
-    @include noto-sans(700);
-    border-radius: 0 20px;
-    padding: 8px;
-    font-size: 14px;
-    line-height: 24px;
-    letter-spacing: 2px;
-
-    @each $color-key, $color-value in $colors {
-      &.#{$color-key} {
-        color: themed($color-key);
-        background-color: themed($color-value);
-      }
-    }
-  }
-}
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: $primary-variant-3;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 22px;
+  font-family: 'Noto Sans';
+  height: 34px;
```

Comment:
IMO the tag dimension should be set by its parent

Reply:
Normally component from design system must be independent. 
We can also add variables like "size" or  "fullWidth" callable by parent